evaluate/task/test-integration/task.go

```diff
@@ -0,0 +1,61 @@
+package testintegration
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	evaluatetask "github.com/symflower/eval-dev-quality/evaluate/task"
+	tasktesting "github.com/symflower/eval-dev-quality/evaluate/task/testing"
+	"github.com/symflower/eval-dev-quality/language/golang"
+	"github.com/symflower/eval-dev-quality/log"
+	"github.com/symflower/eval-dev-quality/model/symflower"
+	"github.com/symflower/eval-dev-quality/tools"
+	toolstesting "github.com/symflower/eval-dev-quality/tools/testing"
+)
+
+func TestTaskWriteTestsRun(t *testing.T) {
+	toolstesting.RequiresTool(t, tools.NewSymflower())
+
+	validate := func(t *testing.T, tc *tasktesting.TestCaseTaskWriteTests) {
+		t.Run(tc.Name, func(t *testing.T) {
+			resultPath := t.TempDir()
+
+			_, logger := log.Buffer()
```

Comment:
Plz always keep the buffer and print it's content in a `defer` in case `t.Failed()`.

Reply:
:cry: I just moved this code

Reply:
Changed.